force-app/main/default/lwc/geFormWidget/geFormWidget.js

```diff
@@ -40,6 +41,7 @@ export default class GeFormWidget extends LightningElement {
         apiNameFor(DATA_IMPORT_DONATION_DONOR),
         apiNameFor(DATA_IMPORT_ACCOUNT_NAME),
         apiNameFor(DATA_IMPORT_PAYMENT_STATUS),
+        apiNameFor(DATA_IMPORT_STATUS),
```

Comment:
This was done to pass this value down to the Tokenize Card component, I don't think it will have any negative impact on existing functionality 🤞